general/development/php.md

```diff
@@ -0,0 +1,30 @@
+---
+title: PHP
+tags:
+  - Requirements
+  - Support
+  - Releases
+  - PHP
+sidebar_position: -400
+---
+New PHP versions are [released every year](https://www.php.net/supported-versions.php) and come with important improvements and changes compared with previous versions. Moodle tries to support them as soon as possible, always matching them with our own [scheduled release plans](/general/releases).
+
+This page contains the current status of support for every PHP version and Moodle versions. For details, follow also the epic links below that combine all the changes that were made in Moodle to ensure PHP compatibility. *(Note: you must be logged in to tracker to see issues in Epics)*
+
+- PHP 7.0 **can be used with** Moodle 3.0.1, Moodle 3.1 and later releases. It is also the **minimum** supported version for Moodle 3.4. See [Moodle and PHP 7.0 details](https://docs.moodle.org/dev/Moodle_and_PHP_7.0_details) and [MDL-50565](https://tracker.moodle.org/browse/MDL-50565) for details.
```

Comment:
It may be worth putting each PHP version in its own heading. That woudl automatically generate an id on the title which can be referenced in a url, and it woudl add it to the ToC.

It _may_ also be worth adding a Since tag to the section. This will need to be imported near the top of the page:

```
import { Since } from '@site/src/components';
```

And then you can use it as a React element:

```suggestion

## PHP 7.0

<Since
    versions={["3.0.1", "3.1"]}
    issueNumber="MDL-50565"
/>

PHP 7.0 **can be used with** Moodle 3.0.1, Moodle 3.1 and later releases. It is also the **minimum** supported version for Moodle 3.4. See [Moodle and PHP 7.0 details](https://docs.moodle.org/dev/Moodle_and_PHP_7.0_details) and [MDL-50565](https://tracker.moodle.org/browse/MDL-50565) for details.
```

The source for that element is at https://github.com/moodle/devdocs/blob/main/src/components/Since.js, though I'll be rewriting it in TypeScript sometime with fully described props.

Reply:
Ok, interesting! I'm going to try this locally, instead of applying your suggestion, so I see it in action. Thanks!